app/packages/core/src/utils/QueryClientProvider.tsx

```diff
@@ -0,0 +1,23 @@
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
```

Comment:
If you want to add this please adjust the QueryClientProvider in all places.

I would also add this to the context folder and export it as QueryClientProvider.